### DISCUSSION TITLE
Inconsistent last commit time

### DISCUSSION THREAD

**Discussion by 2096779623** [community/community#18351] [Other Feature Feedback, Questions, & Ideas]
In the github homepage, hover the mouse over the repository, and you will see abbreviated information about the repository, which is inconsistent with the repository information.

Example:
![image](https://user-images.githubusercontent.com/57583560/173183438-34d5106f-0fe7-4ac0-a938-0a2ccd7acfa1.png)

![image](https://user-images.githubusercontent.com/57583560/173183465-8fd078bf-3914-4140-998b-efb701d89e1e.png)

**Comment by alizain-c** [comment]
Oh woah! I didn't know it was an issue. I never looked into it haha! 
Perhaps, it's a bug.
